core/src/main/scala/org/apache/spark/partial/BoundedDouble.scala

```diff
@@ -21,5 +21,23 @@ package org.apache.spark.partial
  * A Double value with error bars and associated confidence.
  */
 class BoundedDouble(val mean: Double, val confidence: Double, val low: Double, val high: Double) {
-  override def toString(): String = "[%.3f, %.3f]".format(low, high)
+  override def toString(): String =
```

Comment:
OK, I think this is all good, except I think the `toString` should be left alone. I forgot to mention this. Not that I really expect anyone to depend on the format, but let's leave it since it's a public class.